metasmoke.py

```diff
@@ -83,8 +83,8 @@ def init_websocket():
                     failed_connection_attempts += 1
                     if failed_connection_attempts > MAX_MS_WEBSOCKET_RETRIES:
                         chatcommunicate.tell_rooms_with("debug", "Cannot initiate MS websocket." +
-                                                        "  metasmoke_ws_t is now dead.")
-                        log('warning', "Cannot initiate MS websocket. metasmoke_ws_t is now dead.")
+                                                        "  Manual `!!/reboot` is required once MS is up")
+                        log('warning', "Cannot initiate MS websocket. metasmoke_ws_t is terminating.")
```

Comment:
This log really shouldn't mention `metasmoke_ws_t`. Doing so puts a reverse dependency on the code this is being called from, which is in a different file/class. This function doesn't know anything about how it's being called. What if the Metasmoke class is used in some other way?